dnsleaktest.go

```diff
@@ -34,11 +34,14 @@ func fakePing() int {
 
 	rSubDomainId1 := _random(1000000, 9999999)
 
-	for i := 1; i <= 10; i ++ {
+	initUrl := fmt.Sprintf("https://%d.%d.%s", 0, rSubDomainId1, ApiDomain)
+	for i := 1; i <= 10; i++ {
 		rSubDomainId2 := i
-		initUrl := fmt.Sprintf("https://%d.%d.%s", rSubDomainId2, rSubDomainId1, ApiDomain)
-		http.Get(initUrl)
+		initUrl = fmt.Sprintf("https://%d.%d.%s", rSubDomainId2, rSubDomainId1, ApiDomain)
+		go http.Get(initUrl)
 	}
+	fmt.Println("initUrl: ", initUrl)
```

Comment:
Please remove test output from the code.

Reply:
this line was removed in the next commit which improved concurrency with wait groups. look [all commits](https://github.com/macvk/dnsleaktest/pull/12/files#diff-0e272a0578d7ed36cb4ac44664cbb61933c9ffb42dad89c44c03112e860016acR43)